homeassistant/components/camera/__init__.py

```diff
@@ -699,6 +699,10 @@ async def async_handle_play_stream_service(camera, service_call):
         ATTR_ENTITY_ID: entity_ids,
         ATTR_MEDIA_CONTENT_ID: f"{get_url(hass)}{url}",
         ATTR_MEDIA_CONTENT_TYPE: FORMAT_CONTENT_TYPE[fmt],
+        "stream_type": "LIVE",
+        "media_info": {
```

Comment:
This is specific to Cast. We should check if entity originates from cast using [`entity_sources`](https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity.py#L45) and only pass this info to that.